### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by whit3h4wk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,18 @@ Ensure you have downloaded [Node](https://nodejs.org/en/)
 
 **Make sure your node version is below 17.0.0**
 
-When in the project folder (not src), you'll want to install the devDependencies - and if you don't have it, node - like so:
+You can check your node version with:
+
+```pohershell
```
Comment (whit3h4wk) [inline README.md]: typo

**Review by CarsonBurke — CHANGES_REQUESTED** [review]
please fix the typo. Otherwise it looks great!

**Comment by whit3h4wk** [comment]
Fixed typo

**Comment by CarsonBurke** [comment]
Thanks. Last thing is to merge into Main instead of Dev, but I think I can handle that

**Comment by CarsonBurke** [comment]
Seems like I'll have to make the commit irrespective of the pull request to avoid updating Main to the state of Development

**Pull request closed without merging**